packages/react-dom/src/client/ToStringValue.js

```diff
@@ -19,7 +20,7 @@ export opaque type ToStringValue =
 // around this limitation, we use an opaque type that can only be obtained by
 // passing the value through getToStringValue first.
 export function toString(value: ToStringValue): string {
```

Comment:
I'm wary of the layers here. Something calls `ToStringValue.toString` which itself calls `trustedTypesAwareToString`. It's hard to understand all the callsites and which are intentionally accepting trusted types, vs accidentally because they happen to use this module.

I propose to unify this module and `trustedTypesAwareToString.js` into one. And to be explicit about when we're guaranteed to get a string back vs when we may get a TT object.